Makefile

```diff
@@ -62,6 +62,7 @@ development:
 	- docker-compose rm -f
 	docker-compose up -d && \
 	docker-compose exec kong make dev && \
+	docker-compose exec kong chmod -R 777 /kong && \
```

Comment:
oh wait — 777, is that right? Shouldn't it be at least 775 perhaps?

Reply:
ugh I munged two branches together accidentally let me sort that out sorry 😞